bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -230,9 +235,21 @@ def __init(self):
                 self.__crypt, aes_256_ctr_encrypt, aes_256_ctr_decrypt, ffi.NULL):
             self.__raise_from_status()
 
+        if self.__opts.csfle_path is not None:
+            lib.mongocrypt_setopt_set_csfle_lib_path_override(self.__crypt,
+                                                              self.__opts.csfle_path.encode(
+                                                                  "utf-8"))
+
+        if not self.__opts.bypass_encryption:
+            lib.mongocrypt_setopt_append_csfle_search_path(self.__crypt, b"$SYSTEM")
         if not lib.mongocrypt_init(self.__crypt):
             self.__raise_from_status()
 
+        if self.__opts.csfle_required and self.csfle_version is None:
+            raise MongoCryptError("""CSFLE library could not be loaded from either the override 
+                                  path specified {} or from your 
+                                  operating system's dynamic library locator""".format(self.__opts.csfle_path))
```

Comment:
This error message is going to be oddly formatted since multiline comments (`""""""`) retain all the newlines and whitespace. How about:
 ```python
             raise MongoCryptError(
                 "csfle_required=True but the csfle library could not be loaded from"
                 " csfle_path={}".format(self.__opts.csfle_path) +
                 " or the operating system's dynamic library search path")
 ```

Reply:
Done.